presidio-anonymizer/tests/anonymizers/test_anonymizer.py

```diff
@@ -0,0 +1,8 @@
+from presidio_anonymizer.anonymizers import Anonymizer
+
+
+def test_given_anonymizers_list_then_all_classes_are_there():
+    anonymizers = Anonymizer.get_anonymizers()
+    assert len(anonymizers) >= 5
+    for class_name in ["hash", "mask", "redact", "replace", "fpe"]:
+        assert anonymizers.get(class_name)
```

Comment:
what is this asserting? does get return None if it doesn't exist?

Reply:
Done in next PR